### ISSUE TITLE
wGet failed Couldn't resolve host name

### ISSUE THREAD

**Issue by pha3z** [ctubio/Krypto-trading-bot#501]
I was trying to get the bot running on Amazon Linux 2 without success due to a problem with sTunnel that I couldn't resolve.  I switched to Ubuntu on AWS and now I get a basic curl problem instead:

`K build x86_64-linux-gnu 2017-12-21 06:01:52.
K version unknown (zip install).
18:04:41.137165 DB /data/db/K.COINBASE.LTC.USD.db loaded OK.
18:04:41.137468 UI ready over HTTPS on external port 3000.
18:04:41.137757 QP Warrrrning: using default values for Quoting Parameters.
18:04:41.137814 DB loaded 0 historical Trades.
18:04:41.137875 DB loaded 0 STDEV Periods.
18:04:41.137955 DB loaded TBP = 0.00000000 LTC.
18:04:41.183972 CURL Warrrrning: wGet failed Couldn't resolve host name.
18:04:41.184053 CF Errrror: Unable to fetch data from COINBASE symbol "LTC-USD", possible error message: {}.
K crashed with exit code 1.. Respawning..
`
In bash command line, I tried curl and it works fine:
`curl https://api.gdax.com/products/BTC-USD`

I read on another thread that Krypto uses its own built in curl.  Is there any issue there somehow?

Also, AWS supports Suse and Redhat out of the box as well.  Would Krypto work better on one of those distros?  There appears to be a community build Debian for AWS.  It's called Jessie.  Would that provide the most compatibility for Kryptobot?

**Comment by ctubio** [comment]
it could be a few reasons i donno, but mainly is because the url is simply wrong (with a typo somewhere)

can you please confirm you have in the config file `API_HTTP_ENDPOINT=https://api.gdax.com` as the olnly `API_HTTP_ENDPOINT` var enabled? meanwhile i will try to add some more relevant info to the error msg, like the url in use or something

**Comment by pha3z** [comment]
Before I got your reply, I tried Debian and it worked, so I tried Ubuntu again.  This time it worked fine.

I have no way of reviewing the previous config file when it was broken, but I'm pretty certain your suspicion was correct about the config file being misconfigured.  I bet that I uncommented one of the lines to use an endpoint and left the NULL line uncommented also.  Or something to that effect.  I didn't edit any values.  All I did was uncomment for coinbase, gdax, and the wss feed and try to comment out the NULL lines.

Verbose output would be helpful.  Reporting what data was read from the config file, for instance, would alert users to mistakes like this.  Spitting out that information right below the error message would probably be the most useful.

Should I make a pull request to fix this?
